nengo_deeplearning/utils.py

```diff
@@ -292,8 +292,11 @@ def minibatch_generator(inputs, targets, minibatch_size, shuffle=True,
     rng : :class:`~numpy:numpy.random.RandomState`, optional
         random number generator
 
-    Yields
-    ------
+    Returns
+    -------
+
+    A generator that yeilds:
```

Comment:
Hm, sphinx raised a warning for me that `numpydoc` didn't understand it. Do you not get a warning? Maybe I'm using an older version of `numpydoc`?